lib/components/TabView/TabContent.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+
+import styled from "styled-components";
+
+import { TabViewProps } from "./TabView.types";
+
+export const TabContentWrapper = styled.div<Pick<TabViewProps, "bgColor">>`
+  min-height: 100px;
```

Comment:
this is also very limiting, the styling of this component should be as minimal as possible and let the lib consumer dictate the rules for rendering the content

Reply:
No problem removing the min-height or padding, but the border-radius is defined in Figma and I thought it was important to "enforce" this design aspect on the component. I feel sometimes there is a conflict between having a "open/generic UI library" vs enforcing the Figma styles. Maybe we could have a meeting about this with @cld-santos ? I mean, if we let everything "open to who's coding", then we make it more difficult to use the component, since the dev has to check Figma all the time to see the correct styles, icons, etc.